bin/install-prereqs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Install linters
+go get -u github.com/alecthomas/gometalinter
+gometalinter --install --vendored-linters
+
+# Install buildifier BUILD file validator
+go get -u github.com/bazelbuild/buildifier/buildifier
```

Comment:
It will be used in another check file. We want to add that check for auth later. Well, I can remove it since it's not used right now, I guess.